tests/Integration/BUILD.bazel

```diff
@@ -238,21 +222,53 @@ php_gapic_library(
     # For the IAM mixin.
     service_yaml = "apis/kms/v1/cloudkms_test_mixins_v1.yaml",
     deps = [
-        #        ":iam_php_proto",
         ":kms_php_proto",
-        #        "@com_google_googleapis//google/cloud/location:location_php_proto",
     ],
 )
 
 php_gapic_assembly_pkg(
     name = "google-cloud-kms-v1-php",
     deps = [
-        ":iam_php_grpc",
+        #":iam_php_grpc",
         ":kms_php_gapic",
         ":kms_php_grpc",
         ":kms_php_proto",
-        ":kms_proto",
-        "@com_google_googleapis//google/cloud/location:location_php_grpc",
-        "@com_google_googleapis//google/cloud/location:location_php_proto",
+        #"@com_google_googleapis//google/cloud/location:location_php_grpc",
```

Comment:
Can you include a comment as to why these are commented out? Like what was done later on in the service yaml.